Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```diff
@@ -118,6 +118,7 @@ script:
       defaultValue: 'All'
     - description: Comma-separated list of mailboxes/distribution groups to exclude when you use the value "All" for the exchange_location parameter.
```

Comment:
please add a - additionalinfo: 
describing why this field is deprecated.
something like: Deprecated since Microsoft is supporting it only on-premise...